parm/use_cases/model_applications/convection_allowing_models/GridStat_fcstHRRR_obsPracPerfect_SurrogateSevereProb.conf

```diff
@@ -84,22 +84,23 @@ FCST_GRID_STAT_INPUT_DATATYPE = NETCDF
 GRID_STAT_CONFIG_FILE = {CONFIG_DIR}/GridStatConfig_ss_prob
 
 [dir]
-PARM_BASE = /home/kalb/surrogate_severe/METplus/parm
+INPUT_BASE = /d1/projects/METplus/METplus_Data
+OUTPUT_BASE = /d1/METplus_Data
 
```

Comment:
Do you want to define these here or in a system.conf file? I think what we have been doing is also using a system.conf file which lets you move INPUT_BASE and OUTPUT_BASE around without killing the script.

Reply:
I agree that you should remove PARM_BASE, INPUT_BASE and OUTPUT_BASE from this config. PARM_BASE gets set automatically when you run, so setting it here wont do anything. If the user sets INPUT_BASE and OUTPUT_BASE in their default configs, then this will cause the use case to fail unless they are on the machine you used.